### ISSUE TITLE
Feedback on Health App

### ISSUE THREAD

**Issue by MtRainierUTD** [TeamSSRI/teamssri.github.io#8]
**Proposal Name:** Health App

**Feedback:** This is a really well-thought-out proposal with a deep understanding of the target demographic and possible routes for expansion. My three main concerns:
- Security around collecting health data: I'll freely admit that cybersecurity is not my area of expertise. That being said, if we'd be collecting anything personal health information, that's something we'd need to really be careful about from (at minimum) an ethics perspective.
- AI: AI has the tendency to hallucinate. While most cases are relatively harmless (if someone gets the wrong date for when the Titanic sank, that's misinformation but at the end of the day isn't directly materially a large problem), health advice is where that could go incredibly wrong. It does not help that training data most likely includes unsourced and unscientific blogs and not all just medical journals. If something goes wrong, the best case scenario is that the app recommends a vegan and carnivore diet simultaneously. Realistically, based off of the amount of data it has from the internet, there's a decent chance it could be mean to fat people in an unhelpful (or possibly even medically dangerous) way instead of having real sound advice that a potential customer could benefit from. Worst case scenario, it starts recommending genuinely dangerous routines - and we might not know because we didn't directly program the answer.
- Potential legal issues: While health cybersecurity and AI are legal minefields in their own rights, we could run into legal issues even if we just were recommending advice without AI or without collecting data. There are laws that exist about who's allowed to be paid to give nutritional advice, and while there's a chance that Texas happens to have really relaxed laws, I wouldn't count on it. I'm sure this would only compound if AI was involved - ChatGPT can't have a certificate in nutrition studies. 
In short, while this is a really well thought-out proposal in many ways, I'm really concerned about the potential ethical and legal questions that could come from this app.
